examples/smbpasswd.py

```diff
@@ -122,12 +126,22 @@ def parse_args():
 	parser.add_argument('target', action='store', help='[[domain/]username[:password]@]<targetName or address>')
 	parser.add_argument('-ts', action='store_true', help='adds timestamp to every logging output')
 	parser.add_argument('-debug', action='store_true', help='turn DEBUG output ON')
+
 	group = parser.add_mutually_exclusive_group()
 	group.add_argument('-newpass', action='store', default=None, help='new SMB password')
-	group.add_argument('-newhashes', action='store', default=None, metavar='LMHASH:NTHASH', help='new NTLM hashes, format is LMHASH:NTHASH '
-                                                                           '(the user will be asked to change their password at next logon)')
+	group.add_argument('-newhashes', action='store', default=None, metavar='LMHASH:NTHASH', help='new NTLM hashes, format is LMHASH:NTHASH (the user will be asked to change their password at next logon)')
+	group = parser.add_mutually_exclusive_group()
+	group.add_argument('-oldpass', action='store', default=None, help='old SMB password')
+	group.add_argument('-oldhashes', action='store', default=None, metavar='LMHASH:NTHASH', help='old NTLM hashes, format is LMHASH:NTHASH (the user will be asked to change their password at next logon)')
+
```

Comment:
- _oldpass_: Can be obtained through _target_ flag ([smbpasswd.py#L155](https://github.com/fortra/impacket/blob/bbcd2d8f6bfeb7e0eb046f0fe742c689d4d03bda/examples/smbpasswd.py#L155))
- _oldhashes_: Can be obtained through _hashes_ flag ([smbpasswd.py#L169-171](https://github.com/fortra/impacket/blob/bbcd2d8f6bfeb7e0eb046f0fe742c689d4d03bda/examples/smbpasswd.py#L169-L171))
- Mutually exclusive password/hashes: SMBPasswd already prioritizes hashes over username/password ([smbpasswd.py#L169-L177](https://github.com/fortra/impacket/blob/bbcd2d8f6bfeb7e0eb046f0fe742c689d4d03bda/examples/smbpasswd.py#L169-L177))